server/controllers/Imap-connection.js

```diff
@@ -49,58 +46,67 @@ const mails = (
             msg.on('body', (stream, info) => {
               stream.pipe(parser);
               parser.on('end', (parsedMail) => {
-                getMailsCallback(JSON.stringify(parsedMail));
+                cb(JSON.stringify(parsedMail));
               });
             });
           });
           f.once('error', (Err) => {
```

Comment:
follow the camelCase convention.
```suggestion
          f.once('error', (imapError) => {
```

Reply:
I wrote it this way to distinguish it from other error arguments.

Reply:
I understand. But there are better and more descriptive ways to do so.